src/mesido/esdl/asset_to_component_base.py

```diff
@@ -72,8 +72,9 @@ def get_internal_energy(asset_name, carrier):
         logger.warning(
             f"Neither gas or hydrogen was used in the carrier " f"name of pipe {asset_name}"
         )
-        internal_energy = 46.0e6  # natural gas at about 8 bar
-    return internal_energy  # we use gram per second and coolprop gives results in kJ per kg
+        # TODO: resolve heating value (default value below) vs internal energy values (above)
+        internal_energy = 46.0e6  # natural gas at about 1 bar [J/kg] heating value
+    return internal_energy  # [J/kg]
 
 
 def get_density(asset_name, carrier):
```

Comment:
why are we passing asset_name to this method, it is not used, same for get_internal_energy

Reply:
It is used in the warning information, and I think it should stay for this purpose to give proper info about the warning

Reply:
Ahh yes I missed that, sorry!